apps/web/src/configs/mui.ts

```diff
@@ -0,0 +1,2 @@
+'use client';
+export * from '@mui/material';
```

Comment:
new file at end of file

Reply:
What do you mean?

Reply:
git suggests to have an empty line at the end of each file. Text comparissons are smother this way.

Reply:
done